editor/src/components/canvas/canvas-strategies/strategies/reparent-strategy-helpers.ts

```diff
@@ -954,12 +976,24 @@ export function getAbsoluteReparentPropertyChanges(
 export function getStaticReparentPropertyChanges(
   newPath: ElementPath,
   targetOriginalStylePosition: CSSPosition | null,
+  targetOriginalDisplayProp: string | null,
+  convertToInline: 'row' | 'column' | 'do-not-convert',
```

Comment:
This type looks to be in a bunch of places, perhaps it should be a separately defined type?

Reply:
I decided to leave this one type alone, there's a follow up task to rename this to horizontal | vertical anyways